shared/actions/login/saga.js

```diff
@@ -456,7 +483,11 @@ function* handleProvisioningError(error): Generator<any, void, any> {
 function* loginFlowSaga(usernameOrEmail, passphrase): Generator<any, void, any> {
   // If there is passphrase, use that.
   const passphraseSaga = passphrase
-    ? onBackSaga => () => EngineRpc.rpcResult({passphrase: passphrase.stringValue(), storeSecret: false})
+    ? onBackSaga => () =>
+        EngineRpc.rpcResult({
+          passphrase: passphrase ? passphrase.stringValue() : 'NEVER HAPPENS',
```

Comment:
flow thinks this could turn null in the meantime so we just add the extra fallback

Reply:
it's because function args are the same as let statements I think